src/Reducers/userReducer.js

```diff
@@ -1,10 +1,10 @@
-export const userReducer = (state = {}, action) => {
+export const userReducer = (state = {loggedOut: true}, action) => {
```

Comment:
Is using 'loggedOut: true' in this way frowned up to check if a user is logged in on line 8 of MovieDetails.js?? @robbiejaeger